src/app/rosetta/lib/block.ml

```diff
@@ -287,7 +287,16 @@ WITH RECURSIVE chain AS (
 
   module User_commands = struct
     module Extras = struct
-      type t = {fee_payer: string; source: string; receiver: string}
+      (* TODO: A few of these actually aren't used; should we leave in for future or remove? *)
```

Comment:
We could remove `status` and `created_token` entirely from this file now as they're not used